react-components/src/components/Widgets/WindowWidget.tsx

```diff
@@ -27,7 +27,7 @@ type WindowWidgetProps = {
   subtitle?: string;
   header?: string;
   type?: string;
-  openInNewTab?: ReactNode;
+  customComponent?: ReactNode;
```

Comment:
I think this should be called something like `titleBarContent`, just to make it clear where the component will appear